libmscore/fret.h

```diff
@@ -21,43 +21,139 @@ class StringData;
 class Chord;
 class Harmony;
 
+// Keep this in order - not used directly for comparisons, but the dots will appear in
+// this order in fret multidot mode. See fretproperties.cpp.
+enum class FretDotType : signed char {
```

Comment:
why signed char when we don't use negative numbers?

Reply:
Force of habit I guess... it really makes no difference

Reply:
other than documenting something (the need of negative values) that isn't true...

Reply:
I mean, in the context of this it doesn't matter whether it's a signed enum or an unsigned enum. There's no performance improvement, and it doesn't make it more confusing making it signed.